src/deepsparse/v2/operators/operator.py

```diff
@@ -54,14 +56,18 @@ def has_output_schema(cls) -> bool:
     def __call__(
         self,
         *args,
+        inference_state: InferenceState,
```

Comment:
I recommend using `\` operator introduced in python3.8 https://peps.python.org/pep-0570/#syntax-and-semantics